fiftyone/utils/sam2.py

```
@@ -207,133 +183,81 @@ class SegmentAnything2ImageModel(fosam.SegmentAnythingModel):
+        self._sam_predictor.processor.set_image_batch(images)
 
-    def _forward_pass_boxes(self, imgs):
-        sam2_predictor = self._load_predictor()
-        self._output_processor = fout.InstanceSegmenterOutputProcessor(
-            self._curr_classes
+        point_coords = imgs.get("point_coords")
+        point_labels = imgs.get("point_labels")
+        boxes = imgs.get("boxes")
+        mask_inputs = imgs.get("mask_inputs")  # Not used currently
+        multimask_output = (
+            True if (boxes is None and point_coords is not None) else False
```

Ruff (0.15.9)

[warning] Use `bool(...)` instead of `True if ... else False`

Replace with `bool(...)

(SIM210)